login_KA.feature

```diff
@@ -0,0 +1,144 @@
+    And user fills invalid password in "password field"
+    And user clicks "<next> button"
+    Then user sees an error message and "password field" marked witb red color
+    Examples:
+    | email                          |  | password       |
+    | anna.kalinina@quantori.academy |  | wrong_password |
+
+Scenario: 1_2.2 Log in with invalid login
+    Given user has a google account
+    When user fills a invalid login in "login field"
+    And user clicks "<next> button"
+    Then user sees an error message and "login field" marked witb red color
+    Examples:
+    | email                              |  
+    | anna.kalinina@quantori_not_academy |  
+
+Scenario: 1_3.1 Log in with valid phone number 
+   Given user has a gmail account with attached phone number to it
+   When user clicks "<change the account> button"
+   And fills phone number in the field
+   And clicks "<next> button"
+   And user fills valid password in the "<password> field"
+   Then user is logged in
+   And user sees the main page
+
+   Examples:
+   | phone        |  | password       |
+   | +79123456789 |  | right_password |
+
+Scenario: 1_3.2 Log in with invalid phone number 
+   Given user has a gmail account with attached phone number to it
+   When user clicks "<change the account> button"
+   And fills invalid phone number in the field
+   And clicks "<next> button"
+   Then user sees error message and "login field" is marked red
+
+   Examples:
+   | phone      |  
+   | 9123456789 |  
+
+Scenario: 1_3.3 Log in with valid phone number and invalid password
+    Given user has a gmail account with attached phone number to it
+    When user clicks "<change the account> button"
+    And fills phone number in the field
+    And clicks "<next> button"
+    And user fills invalid password in the "<password> field"
+    Then user sees error message and "password field" is marked red
+    
+    Examples:
+    | phone        |  | password       |
+    | +79123456789 |  | wrong_password |
+
+Scenario: 1_4.1 Log in using another account, which is not google one
+    Given user has yandex account
+    When user clicks "<change the account> button"
+    And user fills yndex login
+    And clicks "<next> button"
+    Then user seer the error message "google account is not finded" and "login field" marked red
+
+    Examples:
+    | login               |  
+    | viburnumk@yandex.ru |  
+
+Scenario: 1_5.1 Log in with empty login
+    Given user has a google account
+    When user clicks "<change the account> button"
+    And clicks "<next> button"
+    Then user seer the error message "enter email address or phone number" and "login field" marked red
+
+Scenario: 1_5.2 Log in with empty password
+    Given user has a google account
+    When user clicks "<change the account> button"
+    And fills valid login
+    And clicks "<next> button"
+    Then user seer the error message "enter password" and "password field" marked red
+
+Scenario: 1_6.1 Log in with fresh login
```

Comment:
The background states that the user has a Google account and 'Given' step here says the opposite. So maybe it would be better to change the order of the scenarios

Reply:
Thank you for that! I have changed background, actually I have in many tests "Given user has a gmail account"